controllers/multiclusterobservability/multiclusterobservability_controller_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	migrationv1alpha1 "sigs.k8s.io/kube-storage-version-migrator/pkg/apis/migration/v1alpha1"
```

Comment:
reference to this one or reference to the openshift forked repo?

Reply:
Good point, the import path for forked repo is not changed, so we have to replace the repo in `go.mod`

```
replace (
	sigs.k8s.io/kube-storage-version-migrator => github.com/openshift/kubernetes-kube-storage-version-migrator v0.0.3-0.20210302135122-481bd04dbc78
)
```

see: https://github.com/morvencao/multicluster-observability-operator/blob/br_create_storage_version_migration/go.mod#L39